docs/en/Admin-CLI.md

```diff
@@ -37,6 +37,26 @@ UFS URI like `hdfs://<name-service>/`, and not `hdfs://<name-service>/<folder>`.
 
 ## Example Use Cases
 
+### backup
+
+The `backup` command creates a backup of Alluxio metadata.
+
+Back up to the default backup folder
```

Comment:
specify what the backup folder is (or parameter which configures it)?

Reply:
done